setup-github-credentials.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This should be moved into bfc.

Reply:
Moved via https://github.com/presslabs/bfc/pull/3